app/boards/shields/microdox/microdox.dtsi

```diff
@@ -4,62 +4,22 @@
  * SPDX-License-Identifier: MIT
  */
 
-#include <dt-bindings/zmk/matrix_transform.h>
+#include "microdox_common.dtsi"
 
 / {
     chosen {
-        zephyr,display = &oled;
         zmk,kscan = &kscan0;
```

Comment:
You can move this line into `microdox_common.dtsi` as well and get rid of the chosen node. Same for `microdox_v2.dtsi`.